festim/boundary_conditions/fluxes/surface_kinetics.py

```diff
@@ -0,0 +1,168 @@
+from festim import FluxBC
+from fenics import *
+import sympy as sp
+
+
+class SurfaceKinetics(FluxBC):
+    """
+    FluxBC subclass allowing to include surface processes in 1D H transport simulations
+
+    .. math::
+
+        \dfrac{d c_{\mathrm{s}}}{dt} = k_{\mathrm{bs}} c_{\mathrm{m}} \lambda_{\mathrm{abs}} (1 - \dfrac{c_\mathrm{s}}{n_{\mathrm{surf}}})
+        - k_{\mathrm{sb}} c_{\mathrm{s}} (1 - \dfrac{c_{\mathrm{m}}}{n_\mathrm{IS}}) + J_{\mathrm{vs}};
+
+        -D \dfrac{\partial c_{\mathrm{m}}}{\partial x} = -\lambda_{\mathrm{IS}} \dfrac{\partial c_{\mathrm{m}}}{\partial t}
+        - k_{\mathrm{bs}} c_{\mathrm{m}} \lambda_{\mathrm{abs}} (1 - \dfrac{c_\mathrm{s}}{n_{\mathrm{surf}}})
+        + k_{\mathrm{sb}} c_{\mathrm{s}} (1 - \dfrac{c_{\mathrm{m}}}{n_\mathrm{IS}}),
+
+    where :math:`c_{\mathrm{m}}` is the concentration of solute hydrogen, :math:`c_{\mathrm{s}}` is the surface concentration of hydrogen
+    adsorbed on a surface, :math:`\lambda_{\mathrm{abs}}=n_{\mathrm{surf}}/n_{\mathrm{IS}}`
+
+    Reference: E.A. Hodille et al 2017 Nucl. Fusion 57 056002
```

Comment:
If the model isn't strictly equal to this paper maybe we can write instead of "reference", "For more details see:"

We could also add [this reference](https://www.sciencedirect.com/science/article/pii/S2352179120300272) too